### PR TITLE
Updating tables according to the latest DB_diagram

### LINKED ISSUE #184: Databases: Migrations and Seeding for the new DB diagram
We made changes on the DB Diagram and therefor we need to change the migrations and seeding:

New version (same link) - https://dbdiagram.io/d/5fb801913a78976d7b7cc7d8
Old version - https://dbdiagram.io/d/5fd7a59d9a6c525a03bb052c

Please check the new version and correct the Migrations and the Seeding.

### PR DESCRIPTION
**Pull request by Elia-Git**
# Description

This PR is aimed to update our database tables and seed files according to the latest  DB_diagram version. Link to the latest db_diagram https://dbdiagram.io/d/5fb801913a78976d7b7cc7d8
Fixes #184 

# How to test?

VS Code and MySQL workbench

# Checklist

- [x] We have performed a self-review of our own code
- [x] We have commented on our code, particularly in hard-to-understand areas
- [x] We  have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/migrations/20201216113500_ add_timezone_in_users.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.table('users', function (table) {
+    table.string('timezone').defaultTo('UTC+1:00').notNullable();
```
Comment (tpetrina) [inline src/server/migrations/20201216113500_ add_timezone_in_users.js:3]:
```suggestion
    table.string('timezone').defaultTo('Europe/Copenhagen').notNullable();
```

File: src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+exports.up = function (knex) {
+  return knex.schema.table('events_users', function (table) {
+    table.dropColumn('comment');
+    table.dropColumn('status');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('events_users');
```
Comment (tpetrina) [inline src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js:8]: You would need to create columns `comment` and `status` here.

File: src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+exports.up = function (knex) {
+  return knex.schema.table('feedback', function (table) {
+    table.dropColumn('open_feedback');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('feedback');
```
Comment (tpetrina) [inline src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js:7]: You need to create column `open_feedback`

File: src/server/migrations/20201216124500_drop_2_fk_columns_from_feedback.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.table('feedback', function (table) {
+    table.dropForeign('answered_by');
+    table.dropColumn('answered_by');
+    table.dropForeign('question_id');
+    table.dropColumn('question_id');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('feedback');
```
Comment (tpetrina) [inline src/server/migrations/20201216124500_drop_2_fk_columns_from_feedback.js:10]: You need to recreate columns `answered_by` and `question_id`

**Review by tpetrina — CHANGES_REQUESTED** [review]
Need fixing of some migrations.
Also, fix seeds.

File: src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js (hunk 2 above)
Comment (Elia-Git) [reply src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js:8]: Those columns are in the old version and are supposed to be dropped according to the recent db_diagram.
So why I should create them?

File: src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js (hunk 3 above)
Comment (Elia-Git) [reply src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js:7]: The same as above.

File: src/server/migrations/20201216124500_drop_2_fk_columns_from_feedback.js (hunk 4 above)
Comment (Elia-Git) [reply src/server/migrations/20201216124500_drop_2_fk_columns_from_feedback.js:10]: The same as above

File: src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js (hunk 2 above)
Comment (Elia-Git) [reply src/server/migrations/20201216124300_drop_comment_&-status_from_events_users.js]: Noted and feedback implemented!

File: src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js (hunk 3 above)
Comment (Elia-Git) [reply src/server/migrations/20201216124400_drop_open_feedback_from_feedback.js]: Noted and feedback implemted!

File: src/server/migrations/20201216113500_ add_timezone_in_users.js (hunk 1 above)
Comment (Elia-Git) [reply src/server/migrations/20201216113500_ add_timezone_in_users.js]: Thanks!

**Review by tpetrina — APPROVED** [review]

File: src/server/migrations/20201216124600_add_event_id_into_feedback_fk.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    table.integer('event_id').unsigned().references('id').inTable('events');
+  });
+};
+
+exports.down = function (knex) {
+  return knex.schema.table('feedback', function (table) {
+    table.dropForeign('event_id');
+    table.dropColumn('event_id');
+  });
+};
```
Comment (AkoAyukngat) [inline src/server/migrations/20201216124600_add_event_id_into_feedback_fk.js:12]: Great work here @Elia-Git  and @MaherHussain :)

**Review by AkoAyukngat — APPROVED** [review]

**Pull request merged**